src/rest/gmm_http_server.erl

```diff
@@ -24,7 +24,7 @@ start_server() ->
             %  POST {bulkRequest :: BulkVertexCreationRequestDto} -> void
             {"/graph/vertices/bulk", not_implemented, #{}},
 
-            %%% @todo rest_edges - mostly implemented
+            %%% @todo rest_edges - implemented all except '/bulk'
```

Comment:
Dlaczego to jest powtórzone?

Reply:
Dlatego że jest dokładnie taka sama sytuacja. Tzn. pozostaje zaimplementować zapytanie typu Bulk. Po prostu mam te komentarze dla każdego modułu-handlera